src/aura/disbursementsCreate/disbursementsCreate.cmp

```diff
@@ -0,0 +1,142 @@
+            </h3>
+            <div class="slds-section__content">
+                <div class="slds-grid slds-grid_pull-padded-medium slds-m-bottom_large">
+
+                    <div class="slds-p-horizontal_medium slds-size_1-of-2">
+                        <dl class="slds-list_horizontal slds-wrap">
+                            <dt class="slds-item_label slds-text-color_weak slds-truncate">Contact:</dt>
+                            <dd class="slds-item_detail slds-truncate">{!v.model.fundingRequest.contactName}</dd>
+                            <dt class="slds-item_label slds-text-color_weak slds-truncate">Awarded Amount:</dt>
+                            <dd class="slds-item_detail slds-truncate">
+                                <ui:outputCurrency value="{!v.model.fundingRequest.awardedAmount}"/>
+                            </dd>
+                        </dl>
+                    </div>
+
+                    <div class="slds-p-horizontal_medium slds-size_1-of-2">
+                        <dl class="slds-list_horizontal slds-wrap">
+                            <dt class="slds-item_label slds-text-color_weak slds-truncate">Total Disbursed:</dt>
+                            <dd class="slds-item_detail slds-truncate">
+                                <ui:outputCurrency value="{!v.model.fundingRequest.totalDisbursed}"/>
+                            </dd>
+                            <dt class="slds-item_label slds-text-color_weak slds-truncate">Amount Remaining:</dt>
+                            <dd class="slds-item_detail slds-truncate">
+                                <ui:outputCurrency value="{!v.model.fundingRequest.totalRemaining}"/>
+                            </dd>
+                        </dl>
+                    </div>
+                </div>
+                <div class="slds-grid slds-grid_pull-padded-medium  slds-p-left_large slds-p-right_large">
+
+                </div>
+            </div>
+        </div>
+
+        <div class="slds-section slds-is-open">
+            <h3 class="slds-section__title slds-theme_shade">
+                <span class="slds-truncate slds-p-horizontal_small" title="Basics">Calculate Disbursements</span>
+            </h3>
+            <div class="slds-section__content">
+
+                <div class="slds-grid slds-grid_pull-padded-medium">
+
+                    <div class="slds-p-horizontal_medium slds-size_1-of-2">
+                        <lightning:input name="paymentCount" label="Number of Disbursements" type="number" value="{!v.data.formData.paymentCount}"/>
+                    </div>
+
+                    <div class="slds-p-horizontal_medium slds-size_1-of-2">
+                        <lightning:input label="First Date" name="firstDate" type="date" value="{!v.data.formData.firstDate}"/>
+                    </div>
+                </div>
+
+                <div class="slds-grid slds-grid_pull-padded-medium">
+
+                    <div class="slds-p-horizontal_medium slds-size_1-of-2">
```

Comment:
I think if you add `slds-grid` to this div, you won't need your custom `inline_block` classes on the inputs. Then the other sizing of those inputs should work properly, too.